### PR TITLE
XEP-0065, XEP-0363: Document exposing the service on the user’s domain

### PR DESCRIPTION
**Pull request by linkmauve**
This practice is quite widespread already, and shouldn’t have any
overhead given the domain’s disco#info should already be cached by
XEP-0115/XEP-0390 in the <stream:features/>.

### CODE REVIEW

**Comment by horazont** [comment]
@iNPUTmice Please comment for the change to XEP-0363.

File: xep-0363.xml [hunk 1]
```diff
@@ -205,6 +215,40 @@
...
+  <p>Alternatively, the upload service may be exposed directly on the Requester’s server. In which case, the Requester should use Service Discovery on this domain, like for every previous disco#items results.</p>
+  <example caption='Requester Sends Service Discovery Request to Server'><![CDATA[
+<iq from='requester@example.com/foo'
+    id='a58f1cc2'
+    to='example.com'
+    type='get'>
+  <query xmlns='http://jabber.org/protocol/disco#info'/>
+</iq>
+]]></example>
+  <p>The server returns its information and the Requester inspects it to determine if it contains an identity of category "proxy" and type "bytestreams".</p>
```
Comment (lovetox) [inline xep-0363.xml]: Copy paste error, `proxy`, `bytestream`

**Comment by horazont** [comment]
Rejected by council due to wording, author of the other affected XEP does not reply.

**Comment by lockywolf** [comment]
I would appreciate not needing a separate DNS entry and a tls setting for each component. It would be a good idea.

File: xep-0065.xml [hunk 2]
```diff
@@ -271,6 +281,32 @@
     <feature var='http://jabber.org/protocol/bytestreams'/>
   </query>
 </iq>
+]]></example>
+  <p>Alternatively, the proxy may be exposed directly on the Requester’s server. In which case, the Requester should use Service Discovery on this domain, like for every previous disco#items results.</p>
```
Comment (Flowdalic) [inline xep-0065.xml:285]: nit: a quick grep in the XEPs repo gives the impression that 'requester' is usually written lowercase

**Pull request closed without merging**